src/api/app/models/package.rb

```diff
@@ -1034,6 +1035,14 @@ def remove_linked_packages
     BackendPackage.where(links_to_id: self.id).delete_all
   end
 
+  def remove_devel_packages
+    Package.where(develpackage: self).each do |devel_package|
+      devel_package.develpackage = nil
```

Comment:
On Montag, 19. Oktober 2015, 01:35:26 CEST wrote Henne Vogelsang:
> > @@ -1034,6 +1035,14 @@ def remove_linked_packages
> >      BackendPackage.where(links_to_id: self.id).delete_all
> >    end
> >  
> > +  def remove_devel_packages
> > +    Package.where(develpackage: self).each do |devel_package|
> > +      devel_package.develpackage = nil
> 
> No permission check and nothing? Hm...

yes, it was always that way. The $other maintainer had always the right
to remove this. Just that we had a broken database now after some refactor
submissions.

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany